app/lib/submission_builder/ty2022/states/ny/documents/rtn_header.rb

```diff
@@ -0,0 +1,85 @@
+module SubmissionBuilder
```

Comment:
hopefully i interpreted the xml spec right. it seems like there's a rtnHeader element allowed under the composition>forms section of the NY state data that's separate from the top level return header. this is the start of an implementation for it with just the required fields and the banking info if present